application/src/port.rs

```diff
@@ -1,5 +1,5 @@
-pub use crate::projections::TodoListProjection;
-pub use domain::{todolist::TodoList, todolist_event::TodoListEvent};
+pub use crate::projection::TodoListProjection;
+pub use domain::{todolist_event::TodoListEvent, todolist_state::TodoList};
 use framework::*;
```

Comment:
**Suggestion:** Consider using a more specific import for the `framework::*` module instead of importing all its contents. [best practice]
```suggestion
use framework::{AnyResult, Error};
```